console/calling_commands.rst

```diff
@@ -29,15 +29,14 @@ method)::
 
         protected function execute(InputInterface $input, OutputInterface $output): int
         {
-            $command = $this->getApplication()->find('demo:greet');
+            $greetInput = new ArrayInput([
+                // the command name is passed as first argument
+                'command_name' => 'demo:greet',
```

Comment:
I think it must be
```
$greetInput = new ArrayInput([
                // the command name is passed as first argument
                'command' => 'demo:greet',
...
```
or
```
$greetInput = new ArrayInput([
                // the command name is passed as first argument
                'demo:greet',
...
```
or am I wrong?

Reply:
See https://github.com/symfony/symfony/blob/6.4/src/Symfony/Component/Console/Application.php#L246 for example.

Reply:
But you are right @maxbeckers, given that https://github.com/symfony/symfony/blob/6.4/src/Symfony/Component/Console/Input/InputInterface.php#L30C24-L30C24 is used in https://github.com/symfony/symfony/blob/6.4/src/Symfony/Component/Console/Application.php#L1122, no matter the name of the argument.

Reply:
I just did the test:
```
<?php

namespace App\Command;

use Symfony\Component\Console\Attribute\AsCommand;
use Symfony\Component\Console\Command\Command;
use Symfony\Component\Console\Input\ArrayInput;
use Symfony\Component\Console\Input\InputArgument;
use Symfony\Component\Console\Input\InputInterface;
use Symfony\Component\Console\Input\InputOption;
use Symfony\Component\Console\Output\OutputInterface;
use Symfony\Component\Console\Style\SymfonyStyle;

#[AsCommand(
    name: 'app:test',
    description: 'Add a short description for your command',
)]
class TestCommand extends Command
{

    protected function execute(InputInterface $input, OutputInterface $output): int
    {
        $arguments = new ArrayInput([
            'command_name' => 'list',
        ]);
        $this->getApplication()->doRun($arguments, $output);

        return Command::SUCCESS;
    }
}
```
Results in an error that command name is not a valid argument:
![Screenshot 2023-08-18 094336](https://github.com/symfony/symfony-docs/assets/11738128/2b1fa533-eda1-4ef6-bcd1-acb2010b5bb5)

Reply:
Ah, then maybe we need to fix the line I linked in Symfony 🤔.
Thanks anyway, I'll update the PR accordingly.

Reply:
Ok I linked the argument of the help method 😅, this line is a better example: https://github.com/symfony/symfony/blob/6.4/src/Symfony/Component/Console/Application.php#L258.

Reply:
Fixed, thanks again.